### ISSUE TITLE
Release discrepancy 

### ISSUE THREAD

**Issue by aqeelat** [recharts/react-smooth#107]
In the releases page:
1. release v4.0.3 is assigned tag v4.0.2
2. tag 4.0.3 is missing, so there are a lot of changes not documented (https://github.com/recharts/react-smooth/compare/4.0.3...v4.0.4)
3. release v4.0.4 is not the same as branch 4.0.4. as there are 4 commits missing (https://github.com/recharts/react-smooth/compare/v4.0.4...4.0.4)

**Comment by ckifer** [comment]
Yep I botched the release a bit. I had to unpublish 4.0.3 because it would have broken the world. 

There are not a lot of changes undocumented, everything that has changed is correct as it stands in the release notes

**Comment by ckifer** [comment]
1. updated 4.0.3 release to have a 4.0.3 tag
2. added 4.0.3 tag - the changes are documented in the change release notes
3. master should be even with the 4.0.4 tag now. Deleting the branch


Going to close this

**Comment by ckifer** [comment]
explained a bit more here sorry for the confusion https://github.com/recharts/react-smooth/releases/tag/v4.0.4
